src/jdk.incubator.concurrent/share/classes/jdk/incubator/concurrent/StructuredTaskScope.java

```diff
@@ -222,10 +222,45 @@
  *   scope "B" is the parent of the nested task scope "C".
  * </ul>
  *
- * <p> The tree structure supports confinement checks. The phrase "threads contained in
- * the task scope" in method descriptions means threads started in the task scope or
- * descendant scopes. {@code StructuredTaskScope} does not define APIs that exposes the
- * tree structure at this time.
+ * The <i>descendants</i> of a task scope that are child task scopes that it is a parent
```

Comment:
There is a typo in the update (my fault).
```suggestion
 * The <i>descendants</i> of a task scope are the child task scopes that it is a parent
```